infra/aws/terraform/security-eng-account/cloudtrail.tf

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+resource "aws_cloudtrail" "organizational_trail" {
+  provider                   = aws.audit
+  name                       = local.cloudtrail_bucket_name
+  s3_bucket_name             = aws_s3_bucket.cloutrail_logs.id
+  is_multi_region_trail      = true
+  is_organization_trail      = true
+  enable_log_file_validation = true
+
```

Comment:
Could add
```hcl
  include_global_service_events = true
```
(it's default, so this is being explicit)

Reply:
Done.